projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -176,7 +176,9 @@ const BarChart: FC< BarChartProps > = ( {
 			style={ {
 				width,
 				height,
-				position: 'relative',
+				display: 'flex',
+				flexDirection:
+					showLegend && legendAlignmentVertical === 'top' ? 'column-reverse' : 'column',
```

Comment:
Given that the default values are set via CSS I wonder if using a className to only change the flex direction might be better

```js
className={ clsx(
				'bar-chart',
				styles[ 'bar-chart' ],
				{
					[ styles[ 'bar-chart--legend-top' ] ]: showLegend && legendAlignmentVertical === 'top',
				},
				className
			) }
```

```css
.bar-chart {
	display: flex;
	flex-direction: column;

	svg {
		overflow: visible;
	}

	&-legend {
		margin-top: 1rem;
	}

	&--legend-top {
		flex-direction: column-reverse;
	}
}
```

Reply:
I think it would be worth adding a `gap` to the column too, to control the spacing

![Screenshot 2025-06-26 at 1 47 06 PM](https://github.com/user-attachments/assets/54dcf3db-17ab-457d-a5b5-e2fad4b871e4)